src/bloqade/submission/ir/task_specification.py

```diff
@@ -51,6 +68,15 @@ def __hash__(self):
         return hash((RabiFrequencyAmplitude, self.global_))
 
     def discretize(self, task_capabilities: QuEraCapabilities):
```

Comment:
please go ahead and change all argument names `task_capabilities` to `device_capabilities` which is more accurate name for what this argument means.